src/modules/DatabaseUtility.js

```diff
@@ -262,6 +262,7 @@ const removeItemById = (id) => {
 
 const getItemById = (id) => {
   const data = dataDb.get("items");
+  console.log({ dataDb });
```

Comment:
@isntlazy - We probably don't want to log the entire dataDb here.

Reply:
sure